src/components/JobListings/components/JobListing/JobListing.tsx

```diff
@@ -11,31 +11,54 @@ export function JobListing({ job, filters }: Props) {
   const tagMarkup = job.tags.map((tag) => {
     return <Tag key={tag} name={tag} />;
   });
-  const ariaLabel = `job listing ${job.position}`;
+
+  const newAndFeaturedAriaLabelText = () => {
+    const {isNew, isFeatured} = job
+    if(isNew && isFeatured) {
+      return "This job has been recently posted and is featured"
+    }
+    if(isNew) return "This job has been recently posted"
+    if(isFeatured) return 'This job is featured';
+
+    return ""
+  }
 
   return (
     <article
-      aria-label={ariaLabel}
       className="job-listing bg-white p-4 pt-0 md:pt-4 text-left rounded-md shadow-xl shadow-desaturated-dark-cyan/30 md:flex md:items-center md:justify-center md:gap-5"
       key={job.id}
     >
+      <h1 aria-label={`${job.position} for ${job.company}`}></h1>
       <img
+        aria-hidden="true"
         src={`../assets/images/${job.logo}`}
         alt="logo"
         className="-mt-[25px] md:mt-0 w-12 md:w-[80px] md:h-[80px]"
       />
       <div className="flex-[0_1_100%]">
-        <div className="flex items-center gap-[10px] my-2">
-          <div className="text-xs md:text-base pr-5 md:pr-2.5 text-desaturated-dark-cyan font-bold">
+        <div
+          aria-label={newAndFeaturedAriaLabelText()}
+          className="flex items-center gap-[10px] my-2"
+        >
+          <div
+            aria-hidden="true"
+            className="text-xs md:text-base pr-5 md:pr-2.5 text-desaturated-dark-cyan font-bold"
+          >
             {job.company}
           </div>
           {job.isNew && (
-            <span className="text-sm p-2 pb-1 leading-none text-white rounded-full bg-desaturated-dark-cyan ">
+            <span
+              aria-hidden="true"
+              className="text-sm p-2 pb-1 leading-none text-white rounded-full bg-desaturated-dark-cyan "
+            >
               NEW!
             </span>
           )}
           {job.isFeatured && (
-            <span className="text-sm p-2 pb-1 leading-none text-white rounded-full bg-black ">
+            <span
+              aria-hidden="true"
```

Comment:
Were announcing the message at the grouping level and we can hide the additional elements from screen reader